cmd/up/up.go

```diff
@@ -422,6 +425,11 @@ func (up *upContext) activate(autoDeploy, build bool) error {
 			}
 		}
 
+		if version != model.GetCurrentImageTagVersion() {
+			log.Yellow("The version of the binary image provided does not match the one used in this version.")
```

Comment:
The Okteto CLI version {version} uses the init container image { model.OktetoBinImage }.